docs/guides/backup_security.md

```diff
@@ -1,43 +1,46 @@
-# Backup and security
+# Backup and Security
 
 Security Checklist
 
-- How to backup your account
-- How to restore from a backup
-- How to export a user's Stronghold
-- How to rekey a Stronghold/password rotation
-- Do's and don'ts
+- How to backup your account.
+- How to restore from a backup.
+- How to export a user's Stronghold.
+- How to rekey a Stronghold/password rotation.
+- Do's and don'ts.
 
-## Security checklist
+## Security Checklist
 
-- [ ] I use Stronghold
-- [ ] I use a strong password (32 character length, Shannon Entropy ~ 4.0) for encrypting the stronghold
-- [ ] I rotate the stronghold password on a regular basis
-- [ ] I create a daily backup of the stronghold.snapshot file
-- [ ] I keep a secure history of passwords used (for recovery)
-- [ ] I use a secure password management service that integrates with the server
-- [ ] I use a linux based server (best memory security)
-- [ ] My server is isolated behind a DMZ
+- [ ] I use Stronghold.
+- [ ] I use a strong password (32 character length, Shannon Entropy ~ 4.0) for encrypting the stronghold.
+- [ ] I rotate the stronghold password on a regular basis.
+- [ ] I create a daily backup of the stronghold.snapshot file.
+- [ ] I keep a secure history of passwords used (for recovery).
+- [ ] I use a secure password management service that integrates with the server.
+- [ ] I use a linux based server (best memory security).
+- [ ] My server is isolated behind a DMZ.
 
-## How to backup your account
+## How to Backup Your Account
 
 A simple copy of the stronghold.snapshot file works as a backup (e.g. a daily cronjob rsync/scp with a datetime suffix for example).
 
-## How to restore from a backup
+## How to Restore From a Backup
 
 Simply place a snapshot file in your directory that wallet.rs expects.
```

Comment:
```suggestion
Simply place a snapshot file in the directory that wallet.rs expects. 
```

It'd be good to specify in which file this is configured.  Something like, "You can find the snapshot directory in the X config file under X section"